packages/completer/src/tokens.ts

```diff
@@ -138,6 +138,131 @@ export interface ICompletionProvider<
   ): boolean;
 }
 
+/**
+ * Describes how an inline completion provider was triggered.
+ * @alpha
```

Comment:
should we use @alpha or @experimental to align with the new API ts doc?

Reply:
If I recall correctly, this is exactly what is mentioned in [tsdoc](https://tsdoc.org/pages/tags/experimental/):

> Same semantics as @\beta, but used by tools that don't support an @\alpha release stage.

Some tools have _experimental_ and not _alpha_/_beta_, other the other way around (like [API extractor](https://api-extractor.com/pages/tsdoc)). I think _experimental_ was even the only one supported by tsdoc in the past.

Reply:
Would you prefer me to change the existing tag comments to:

```
    * @alpha 
    * @experimental 
```

or are they good as they are?

Reply:
Let's keep it like this as `alpha` / `beta` is clearer than `experimental` in term of API stability expectation.